src/lib/HST/Options.hs

```diff
@@ -2,14 +2,24 @@
 --   options.
 
 module HST.Options
-  ( Options(..)
+  ( Frontend(..)
+  , Options(..)
   , defaultOptions
+  , frontendMap
     -- * Command Line Option Parser
   , optionDescriptors
   , parseArgs
+  , parseFrontend
   )
 where
 
+import           Data.List                      ( intercalate )
+import           Data.Map.Strict                ( Map )
+import qualified Data.Map.Strict               as Map
+                                                ( fromList
+                                                , keys
+                                                , lookup
+                                                )
```

Comment:
If you use a qualified import, you don't need to list the imports explicitly. We also don't do this for the `Syntax` module for example.

```suggestion
```